### PR TITLE
Add TomatoBar

### PR DESCRIPTION
**Pull request by ivoronin**
## Project URL
https://github.com/ivoronin/TomatoBar

## Category
Productivity

## Description
Pomodoro Technique Timer for macOS with Touch Bar support.

## Checklist

- [x] Edit [applications.json](https://github.com/serhii-londar/open-source-mac-os-apps/blob/master/applications.json) instead of [README.md](https://github.com/serhii-londar/open-source-mac-os-apps/blob/master/README.md).
- [x] Only one project/change is in this pull request
- [x] Screenshots(s) added if any
- [x] Has a commit from less than 2 years ago
- [x] Has a **clear** README in English

### CODE REVIEW

**Comment by serhii-londar** [comment]
@ivoronin Please resolve conflicts.

File: applications.json [hunk 1]
```diff
@@ -7235,6 +7235,24 @@
...
+                "https://raw.githubusercontent.com/Ribtoks/xpiks/master/assets/xpiks-main-window.png"
+            ],
+            "short_description": "Xpiks is a keywording and uploading tool for microstock photographers, illustrators and videographers",
+            "languages": [
+                "cpp"
+            ],
+            "categories": [
+                "Productivity",
+                "Images",
+                "Video"
```
Comment (serhii-londar) [inline applications.json]: Plese use lowercased categories names.

File: applications.json [hunk 2]
```diff
@@ -7222,6 +7222,20 @@
                 "rust"
             ]
         },
+        {
+            "short_description": "Pomodoro Technique Timer for macOS with Touch Bar support.",
+            "categories": [
+                "productivity"
```
Comment (ivoronin) [inline applications.json]: It's in lower case

**Comment by serhii-londar** [comment]
@ivoronin Thank you! 👍

**Pull request merged**